packages/components/radio/src/radio.tsx

```diff
@@ -80,12 +96,18 @@ export const useRadio = <
   const {
     id: _id,
     name,
+    checked: checkedProp,
+    defaultChecked,
     defaultIsChecked,
     isChecked: isCheckedProp,
     value,
     onChange: onChangeProp,
     ...computedProps
   } = useFormControlProps({ id, ...props })
+
+  const checkedPropValue = checkedProp ?? isCheckedProp
+  const _defaultChecked = defaultChecked ?? defaultIsChecked
```

Comment:
I think this is better.
```
checkedProp ??= isCheckedProp
defaultChecked ??= defaultIsChecked
```

Reply:
These variables cannot be redeclared with these names. It will result in an error.

Reply:
I see.It's a Quality error. It was in a wait state for a while and I didn't notice it, but I'll check it.

Reply:
I think we can change `const` to `let`.

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3548/commits/15f131b106a54656c8ab6d02800e1dedbcdffcc6#diff-7c900e8c44fc7d591b52e5a3b32bdcbf8405d2bbf6874bfa9d58e09d88d11661R96